### ISSUE TITLE
New video format - not downloadable?

### ISSUE THREAD

**Issue by lordoffools** [DIGITALCRIMINALS/UltimaScraper#1085]
Looks like OF has been more widely switching to streaming videos (similar to YouTube*) instead of easily downloadable mpegs. These seem to get skipped by the scraper.

Is anyone else encountering this issue?

(YouTube "streaming" downloads are solved by yt-dlp.

**Comment by lordoffools** [comment]
Anyone else seeing this issue?

**Comment by evamay222** [comment]
Not the only one. I checked. Scrapper stops at a certain point and the newer vids cannot be downloaded. Twas fun while it lasted.

**Comment by beanie-dude** [comment]
I assume the new format you're talking about is "blob:"?  It's on videos as well -- it's the DRM content.  It is possible to download them, but you need tools like ffmpeg and the DRM keys.  I haven't tried it yet, but I know other projects claim they can download the DRM content, such as https://github.com/sim0n00ps/OF-DL (though, that particular project doesn't support downloading streams yet)

**Comment by lordoffools** [comment]
Indeed, the blob.

Fansly does it - some of the other Fansly scrapers seem to download those videos now. And yt-dlp has been downloading the YouTube blobs just fine for a while, so I know it's doable.

**Comment by igobytwo** [comment]
> Anyone else seeing this issue?

Yes.  I was wondering why I was suddenly not getting the videos from 1 of the subs - luckily only one sub right now is affected. I couldn't figure out a way to grab the video outside of the scraper.
